google/cloud/security/inventory/inventory_loader.py

```diff
@@ -372,45 +242,68 @@ def _send_email(cycle_time, cycle_timestamp, status, pipelines,
         LOGGER.error('Unable to send email that inventory snapshot completed.')
 
 
-def _configure_logging(configs):
+def _configure_logging(flag_configs):
     """Configures the loglevel for all loggers."""
-    desc = configs.get('loglevel')
+    desc = flag_configs.get('loglevel')
     level = LOGLEVELS.setdefault(desc, 'info')
     log_util.set_logger_level(level)
 
 def main(_):
     """Runs the Inventory Loader."""
-    try:
-        dao = Dao()
-        project_dao = proj_dao.ProjectDao()
-        organization_dao = org_dao.OrganizationDao()
-        bucket_dao = buck_dao.BucketDao()
-        cloudsql_dao = sql_dao.CloudsqlDao()
-        fwd_rules_dao = fr_dao.ForwardingRulesDao()
-        folder_dao = folder_resource_dao.FolderDao()
-    except data_access_errors.MySQLError as e:
-        LOGGER.error('Encountered error with Cloud SQL. Abort.\n%s', e)
+
+    flag_configs = FLAGS.FlagValuesDict()
+    _configure_logging(flag_configs)
```

Comment:
Can we change this function to just take the loglevel flag?

_configure_logging(flags.get('loglevel'))